dom.bs

```diff
@@ -6946,6 +6948,14 @@ steps:
  <a for=Attr>value</a>.
 </ol>
 
+<p>To <dfn>verify attribute value</dfn> given a {{TrustedType}} or string <var>value</var>, an
```

Comment:
```suggestion
<p>To <dfn>verify an attribute value</dfn> given a {{TrustedType}} or string <var>value</var>, an
```
Although why do we even have this algorithm as it directly calls into the Trusted Types algorithm.